zeroex/order_validator.go

```diff
@@ -147,18 +147,16 @@ func (o *OrderValidator) BatchValidate(signedOrders []*SignedOrder) map[common.H
 				results, err := o.orderValidator.GetOrdersAndTradersInfo(opts, params.Orders, params.Signatures, params.TakerAddresses)
 				if err != nil {
 					log.WithFields(log.Fields{
-						"err":            err.Error(),
-						"attempt":        b.Attempt(),
-						"orders":         params.Orders,
-						"takerAddresses": params.TakerAddresses,
+						"err":       err.Error(),
```

Comment:
This key should be `error` instead of `err`. It's going to be really important when we set up log monitoring.

Reply:
Found this same issue in a bunch of places, changed all of them.

Reply:
🙏